jws/signer.go

```diff
@@ -29,7 +29,7 @@ var signerDB map[jwa.SignatureAlgorithm]SignerFactory
 //
 // Unlike the `UnregisterSigner` function, this function automatically
 // calls `jwa.RegisterSignatureAlgorithm` to register the algorithm
-// in the known algorithms database.
+// in the known-algorithms database.
```

Comment:
This looks to me strange, and I couldn't find a single hit against `"known-algorithms"` on Google...

Reply:
It just felt strange for me, I think I did that change because it felt more like a database name that way. Maybe it should be changed to this "in the known algorithms' database."?

Reply:
Perhaps...? :

```suggestion
// in this module's algorithm database.
```

Reply:
Works for me!